py/mpprint.h

```diff
@@ -39,6 +39,12 @@
 #define PF_FLAG_ADD_PERCENT       (0x100)
 #define PF_FLAG_SHOW_OCTAL_LETTER (0x200)
 
+#if MICROPY_PY_IO
+#    define MP_PRINT mp_sys_stdout_print
```

Comment:
Based on the contents of the file you modify, MP_PRINT would be a function.

Also, there're existing code where there's mp_sys_stdout_print vs mp_plat_print dichotomy, what about it? (Should be a separate commit if anything).